spacy/strings.pyi

```diff
@@ -1,21 +1,15 @@
-from typing import Optional, Iterable, Iterator, Union, Any, overload
+from typing import Optional, Iterable, Iterator, Union, Any, Tuple
 from pathlib import Path
 
-def get_string_id(key: Union[str, int]) -> int: ...
-
 class StringStore:
-    def __init__(
-        self, strings: Optional[Iterable[str]] = ..., freeze: bool = ...
-    ) -> None: ...
-    @overload
-    def __getitem__(self, string_or_id: Union[bytes, str]) -> int: ...
-    @overload
-    def __getitem__(self, string_or_id: int) -> str: ...
```

Comment:
It's correct in that we still take `str` and `int` inputs, but I thought it would be better to simply have a single declaration with a `Union` as the type hint.

Reply:
I think the original overloads are clearer / more correct and can probably make typing for downstream code easier.